### PR TITLE
Camel quarkus firewatch base config and zstream expansion

### PR DESCRIPTION
**Pull request by smpsgill**

### CODE REVIEW

File: ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/jboss-fuse-camel-quarkus-openshift-interop-main__camel-quarkus-ocp4.16-lp-interop.yaml [hunk 1]
```diff
@@ -41,11 +43,9 @@ tests:
       FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
```
Comment (calebevans) [inline ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/jboss-fuse-camel-quarkus-openshift-interop-main__camel-quarkus-ocp4.16-lp-interop.yaml:43]: Doesn't this need to change?

**Review by calebevans — CHANGES_REQUESTED** [review]

File: ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/jboss-fuse-camel-quarkus-openshift-interop-main__camel-quarkus-ocp4.16-lp-interop.yaml (hunk 1 above)
Comment (Simranpal) [reply ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/jboss-fuse-camel-quarkus-openshift-interop-main__camel-quarkus-ocp4.16-lp-interop.yaml]: Thanks for catching that, I have updated it now

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**